nvflare/private/fed/app/simulator/simulator.py

```diff
@@ -48,5 +47,7 @@ def parse_args():
 
     simulator = SimulatorRunner(args)
     if simulator.setup():
-        simulator.run()
-    os._exit(0)
+        run_status = simulator.run()
+    else:
+        run_status = 1
+    os._exit(run_status)
```

Comment:
why we use `os._exit` here? should it be `sys.exit`?

Reply:
os._exit() to terminate right away and force exit. sys.exit is not guarantee.

Reply:
But according to the python doc: 
"https://docs.python.org/3/library/os.html#os._exit"

Note The standard way to exit is sys.exit(n). [_exit()](https://docs.python.org/3/library/os.html#os._exit) should normally only be used in the child process after a [fork()](https://docs.python.org/3/library/os.html#os.fork).

Reply:
sys.exit() will not guarantee the process to be terminated. It could result in the process to hang, for example if a resource is not fully released. os._exit() will force the process to be terminated.

Reply:
Got it.

But another question here, if the resource is not fully released, should we terminate the process?

Reply:
we need to terminate the process.